README.md

```diff
@@ -3,7 +3,11 @@
 [PostgreSQL](https://www.postgresql.org/), is a popular open-source object-relational database.
 The `loopback-connector-postgresql` module is the PostgreSQL connector for the LoopBack framework.
 
-<div class="gh-only">For more information, see the <a href="http://loopback.io/doc/en/lb3/PostgreSQL-connector.html">documentation</a>.
+<div class="gh-only">
+The PostgreSQL connector supports both LoopBack 3 and LoopBack 4. For more information, see
+<a href="https://loopback.io/doc/en/lb4/PostgreSQL-connector.html">LoopBack 4 documentation</a>,
+<a href="http://loopback.io/doc/en/lb3/PostgreSQL-connector.html">LoopBack 3 documentation</a>
```

Comment:
These two pages are currently generated from this file, so it's a bit odd to me. I couldn't come up a better place for the info tho. Since we are improving Postgres connector docs for LB4, I think it's fine.